app/controllers/tasks_controller.rb

```diff
@@ -4,42 +4,72 @@ def index
     render :index
   end
 
+  # Display a specific task
   def show
-    @task = Task.find(params[:id])
+    @task   = Task.find(params[:id])
     render :tasks
   end
 
   def new
     @task = Task.new
-
     render :new
   end
 
-  # Add and save a new task to db
+  # Add and save a new task to list
   def create
-    @task = Task.new(task_params[:task])
-    @task.save
-
+    task = Task.new(task_params[:task])
+    task.save
```

Comment:
`#new` and then `#save` is the equivalent of calling `#create` which means you could write this as a single line:

```ruby
Task.create(task_params[:task])
```